[REF] pylint_pr.conf: Enable check for deprecated openerp.osv module and enable check xml-syntax-error.

moylop260 opened the pull request:
- `openerp.osv` module is deprecated for odoo >= 8.0
 - Error of syntax in xml files is important to show.

moylop260 commented:
No. Travis Build | PR | version | result deprecated openerp.osv | result xml-syntax
------------ | ------------- | ------ | ---- | ----
6 | No | 8.0 | Muted | Muted
7 | Yes | master | Enabled | Enabled
8 | Yes | 7.0 | Muted | Enabled

moylop260 commented:
ping

pedrobaeza commented on travis/cfg/travis_run_pylint_pr.cfg:
```diff
@@ -63,5 +67,4 @@ ignore-docstrings=yes
 [IMPORTS]
-deprecated-modules=pdb,pudb,ipdb
-
+deprecated-modules=openerp.osv
```
Are now pdb and company allowed?

moylop260 replied:
Pdb is checked by pylint stable config file.
"Deprecated module" is disabled in pr config file before of this change.
El sep 23, 2015 1:47 AM, "Pedro M. Baeza" <notifications@github.com>
escribió:

> In travis/cfg/travis_run_pylint_pr.cfg
> <https://github.com/OCA/maintainer-quality-tools/pull/243#discussion_r40174715>
> :
>
> > @@ -63,5 +67,4 @@ ignore-docstrings=yes
> >  notes=
> >
> >  [IMPORTS]
> > -deprecated-modules=pdb,pudb,ipdb
> > -
> > +deprecated-modules=openerp.osv
>
> Are now pdb and company allowed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OCA/maintainer-quality-tools/pull/243/files#r40174715>
> .
>

pedrobaeza replied:
But it should be also checked on PR pylint I think

moylop260 replied:
@pedrobaeza 
This is not necesary, because is duplicate the same error in pr.cfg and stable.cfg
But I don't have problem in duplicate the same error then I fixed!
Thanks for you feedback!

pedrobaeza commented:
:+1:

dreispt commented:
:+1:

moylop260 commented:
@pedrobaeza
Sorry, I had forgotten the following [error](https://bitbucket.org/logilab/pylint/issues/362/bug-w0402-not-show-fails-if-module-not-is)
Then I added a folder with all deprecated python modules to simulate that is installed and I added to sys-path of pylint to find it.

For compatibility with pylint cleaners builds and git-hook

Could you do a quick review?

moylop260 commented:
The result was:
![pylint_result](https://cloud.githubusercontent.com/assets/6644187/10086989/18b30d74-62d7-11e5-818c-a397f690a247.png)

moylop260 commented:
Can be merged?

pedrobaeza commented:
Let's wait for another approval as this is on MQT and then I merge

oscarolar commented:
:+1:

moylop260 commented:
@pedrobaeza 
Ok, I got it

Now, Could you merge it?

yvaucher commented on tests/test_repo/broken_deprecated/__openerp__.py:
```diff
@@ -0,0 +1,12 @@
+    'name': 'Broken deprecated module for tests MQT',
+    'license': 'AGPL-3',
+    'author': 'Odoo Community Association (OCA)',
+    'version': '8.0.0.1.0.0',
```
Too many digits here
